### DISCUSSION TITLE
Budibase Sorting Issue

### DISCUSSION THREAD

**Discussion by mathewhealthi** [Budibase/budibase#8605] [Help] [answered]
When I sort a table of numbers, it sorts only based on the first digit. Can someone help understand/fix this issue?

![image](https://user-images.githubusercontent.com/105265941/200756367-33f5ffb0-3f4c-403f-ae8b-86438002521c.png)

**Comment by avivkri** [comment, marked as answer]
Hi @mathewhealthi,

This could be due to the type of the column being Text instead of Number. Please check the data type of the value.

**Reply by melohagan** [reply]
**Table column**
![Screenshot 2022-11-10 at 09 23 11](https://user-images.githubusercontent.com/101575380/201051572-dc9dff0f-cfd4-4a20-afe4-fd7fe6ac060f.png)

**Custom query**
![Screenshot 2022-11-10 at 09 23 58](https://user-images.githubusercontent.com/101575380/201051747-537205c0-07c6-4fb6-91a5-8ee0eaa8e580.png)
*Make sure to click 'Save query' after you select the data type*
